freeipa/src/main/java/com/sequenceiq/freeipa/kerberosmgmt/v1/KerberosMgmtV1Service.java

```diff
@@ -349,7 +360,7 @@ private String getExistingKeytab(String canonicalPrincipal, FreeIpaClient ipaCli
         }
     }
 
-    private static String constructCanonicalPrincipal(String serviceName, String hostName, String realm) {
+    private static String constructPrincipal(String serviceName, String hostName, String realm) {
```

Comment:
`hostName` is a bit misleading as we pass on alias too

Reply:
Unfortunately, we use hostname for fqdn and also we use hostname for hostname in many places. I have renamed the to serverHostNameAlias, but consistent naming everywhere in the code is a massive refactor